com.unity.cinemachine/Editor/Utility/CmCameraInspectorUtility.cs

```diff
@@ -43,6 +43,10 @@ public static void AddCameraStatus(this UnityEditor.Editor editor, VisualElement
                 + "separate objects, not parented to each other.</b>", 
                 HelpBoxMessageType.Error));
 
+            var noBrainMessage = ux.AddChild(InspectorUtility.HelpBoxWithButton(
+                "A CinemachineBrain is required in the scene.", 
+                HelpBoxMessageType.Error, "Add Brain", () => CinemachineMenu.GetOrCreateBrain()));
```

Comment:
NP why not a warning instead of an error? Is it going to crash the editor?

Reply:
Good point, will change.